testing/tests/DevExpress.core/utils.data.tests.js

```diff
@@ -379,6 +379,13 @@ QUnit.test("plain objects are cloned if previous value is null (T521407)", funct
     assert.equal(dataSource1.store, "Store 1");
 });
 
+QUnit.test("non existing multi-level prop", function(assert) {
```

Comment:
I think that this behavior should work well for wrapped values. But the test below fails:

```
QUnit.module("setter with wrapped variables", {
...

QUnit.test("non existing multi-level prop", function(assert) {
    var mockWrapper = variableWrapper.wrap,
        obj = mockWrapper({
            prop: mockWrapper(undefined)
        });

    var setter = SETTER("prop.subProp")
    setter(obj, "New value");
    assert.equal(obj().prop().subProp, "New value")
});
```